core/model/modx/moddashboardwidget.class.php

```diff
@@ -311,26 +313,14 @@ public function getFileChunk($tpl,array $placeholders = array()) {
      * Render the widget content as if it were a Snippet
      *
      * @param string $content
-     *
      * @return string
      */
-    public function renderAsSnippet($content = '')
-    {
-        if (empty($content)) {
-            $content = $this->widget->get('content');
-        }
-        $content = str_replace(['<?php', '?>'], '', $content);
-        $closure = function ($scriptProperties) use ($content) {
-            global $modx;
-            if (is_array($scriptProperties)) {
-                extract($scriptProperties, EXTR_SKIP);
-            }
-
-            return eval($content);
-        };
-
-        return $closure([
+    public function renderAsSnippet($content = '') {
+        if (empty($content)) $content = $this->widget->get('content');
```

Comment:
Please, do not use short syntax in `if` statements (PSR requires brackets). Also, it may produce elusive issues in the future.